onnxruntime/core/providers/vitisai/imp/tensor_proto.h

```diff
@@ -1,10 +1,25 @@
 // Copyright (c) 2023 Advanced Micro Devices, Inc. All rights reserved.
 // Licensed under the MIT License.
 #pragma once
-//
+
 #include "core/common/gsl.h"
 #include "onnx/onnx_pb.h"
+#include "core/providers/dml/DmlExecutionProvider/src/External/D3DX12/d3dx12.h"
+#include "core/providers/dml/DmlExecutionProvider/src/ErrorHandling.h"
```

Comment:
Same here. Please remove absolute paths. Instead, you can configure the CMake to find the headers automatically. I prefer something like this:
```c++
#include "D3DX12/d3dx12.h"
#include "ErrorHandling.h"
```